packages/next/src/server/app-render.tsx

```diff
@@ -1519,6 +1519,7 @@ export async function renderToHTMLOrFlight(
         Component: () => {
           return (
             <>
+              <Component {...props} />
```

Comment:
Maybe leave a comment there indicating that this component should come before fonts and styles?

Reply:
Yeah, I wanted but forgot, thanks +1